library/spdm_requester_lib/libspdm_req_get_certificate.c

```diff
@@ -261,11 +249,6 @@ static libspdm_return_t libspdm_try_get_certificate(libspdm_context_t *spdm_cont
             status = LIBSPDM_STATUS_INVALID_MSG_FIELD;
             goto done;
         }
-        if (chunk_enabled && (spdm_response->remainder_length != 0)) {
```

Comment:
We should use (chunk_enable && (length == 0xffff) && (spdm_response->remainder_length != 0))

Reply:
Thanks. I have changed the code.